README.md

```diff
@@ -1,58 +1,34 @@
-# mia_template_service_name_placeholder
+# React Template walkthrough
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This walkthrough will explain you how to correctly create a microservice that renders a React based webpage from the DevOps Console.
 
-## Available Scripts
+In order to do so, access to [Mia-Platform DevOps Console](https://console.cloud.mia-platform.eu/login), create a new project and go to the **Design** area.  
+From the Design area of your project select _Microservices_ and then create a new one, you have now reached [Mia-Platform Marketplace](https://docs.mia-platform.eu/development_suite/api-console/api-design/marketplace/)!  
+In the marketplace you will see a set of Examples and Templates that can be used to set-up microservices with a predefined and tested function.  
 
-In the project directory, you can run:
+For this walkthrough select the following template: **React Template**.
+Give your microservice the name you prefer, in this walkthrough we'll refer to it with the following name: **react-service**. Then, fill the other required fields and confirm that you want to create a microservice.  
+A more detailed description on how to create a Microservice can be found in [Microservice from template - Get started](https://docs.mia-platform.eu/development_suite/api-console/api-design/custom_microservice_get_started/#2-service-creation) section of Mia-Platform documentation.
 
-### `yarn start`
+In order to access to your new microservice it is necessary to create an endpoint that targets it.  
+In particular, in this walkthrough you will create an endpoint to your microservice *react-service*. To do so, from the Design area of your project select _Endpoints_ and then create a new endpoint.
+Now you need to choose a path for your endpoint and to connect this endpoint to your microservice. Give to your endpoint the following path: **/react**. Then, specify that you want to connect your endpoint to a microservice and, finally, select *react-service*.  
```

Comment:
Maybe should be specified to uncheck: `Support only JSON format on response`, because this endpoint does not expose a json but an html page